module/actor/sheets/base.js

```diff
@@ -141,43 +141,37 @@ export default class ActorSheetSS2e extends ActorSheet {
       .on('click', this._onMinusInitiative.bind(this));
 
     //Create Inventory Item
-    html.find('.item-create').on('click', this._onItemCreate.bind(this));
+    html.find('.item-create').click(this._onItemCreate.bind(this));
 
     // Update Inventory Item
-    html.find('.item-edit').on('click', this._onItemEdit.bind(this));
+    html.find('.item-edit').click(this._onItemEdit.bind(this));
 
     // Delete Inventory Item
-    html.find('.item-delete').on('click', this._onItemDelete.bind(this));
+    html.find('.item-delete').click(this._onItemDelete.bind(this));
 
     //Expand item summary
-    html
-      .find('.item h4.item-name')
-      .on('click', (event) => this._onItemSummary(event));
+    html.find('.item h4.item-name').click(event => this._onItemSummary(event));
```

Comment:
The `click` event handler method has been deprecated since jQuery 3.3 and has been replaced with the `on` function for binding event handlers. This should continue using the `.on('click', ...)` style.